### PR TITLE
Updating new certification logos

### PR DESCRIPTION
**Pull request by thetwopct**
Updates to Artwork repo regarding new certifications

- Added KCSA png file (still hunting down the SVG)
- Changed Prometheus Certifications section to generalised Cloud Native Certifications
- Included new certifications - CAPA, CCA, ICA

Have left room to add further variations once they surface

### CODE REVIEW

**Comment by thetwopct** [comment]
@cjyabraham Have updated this with new SVGs sent through late yesterday. But variants are missing.

**Review by cjyabraham — CHANGES_REQUESTED** [review]
Looks like this row needs an extra column on the left edge:

Screenshot 2023-11-17 at 1 26 07 PM

**Comment by thetwopct** [comment]
Updated

File: examples/other.md [hunk 1]
```diff
@@ -114,84 +120,117 @@ Please note that CNCF prefers the horizontal form of our logo to the stacked ver
...
+        <td><img src="/other/ica/color/ica-icon-color.svg" width="125"></td>
     </tr>
     <tr>
         <th>black</th>
-      <td><img src="/other/prometheus-certified-associate/black/PrometheusCertifiedAssociate-black.png" width="125"></td>
         <td><img src="/other/prometheus-certified-associate/black/PrometheusCertifiedAssociate-black.png" width="125"></td>
+        <td>Soon</td>
+        <td>Soon</td>
+        <td>Soon</td>
+        <td><img src="/other/prometheus-certified-associate/black/PrometheusCertifiedAssociate-black.png" width="125"></td>
```
Comment (cjyabraham) [inline examples/other.md:171]: This should be an SVG

Comment (thetwopct) [reply examples/other.md]: Updated for you. I think it would have been quicker for you to just change it.

**Comment by thetwopct** [comment]
Updated

**Pull request merged**